app/vmagent: fix creating target id if `--promscrape.dropOriginalLabels` flag was used (#4616)
VictoriaMetrics/VictoriaMetrics: docs/CHANGELOG.md

* BUGFIX: [vmctl](https://docs.victoriametrics.com/vmctl.html): interrupt explore procedure in influx mode if vmctl found no numeric fields.
* BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert.html): use RFC3339 time format in query args instead of unix timestamp for all issued queries to Prometheus-like datasources.
* BUGFIX: vmselect: fix timestamp alignment for Prometheus querying API if time argument is less than 10m from the beginning of Unix epoch.

> * BUGFIX: [vmagent](https://docs.victoriametrics.com/vmagent.html): disable all links with target id at the `/targets` page when `--promscrape.dropOriginalLabels` flag was used for `vmagent`. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4597).

Reviewer: ```suggestion
* BUGFIX: [vmagent](https://docs.victoriametrics.com/vmagent.html): hide `debug` links on the `/targets` page if `--promscrape.dropOriginalLabels` is enabled. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4597).
```
